src/components/TextWrapper.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      margin: '0 0',
+    },
+    items: {
+      display: 'flex',
+      flex: '1',
+      justifyContent: 'space-between',
+      margin: '2px 0px',
+    },
+    titleText: {
+      color: '#6A6A6A',
+      margin: '10px 0',
+    },
+    bold: {
+      fontWeight: 700,
+    },
+  }),
+);
+
+interface TextWrapperProps {
+  title?: string;
+  numbers: number[];
+  labels: string[];
+  units?: string[];
+  color?: 'inherit' | 'primary' | 'textSecondary';
+  bold?: boolean;
+}
+
+export default function TextWrapper(props: TextWrapperProps): JSX.Element {
+  const classes = useStyles();
+
+  const getLabeledNumber = (number: number, label: string, index: number) => {
+    return (
+      <div key={index} className={classes.items}>
```

Comment:
It's better if indices aren't used as keys. See https://reactjs.org/docs/lists-and-keys.html#keys and https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318 . Maybe we could use label instead if that's unique?